lib/outbound/odata.js

```diff
@@ -345,7 +345,7 @@ const rowStreamToOData = (fields, table, domain, originalUrl, query, inStream, t
       const totalCount = (tableCount != null) ? tableCount : counted;
       const nextUrl = nextUrlFor(limit, offset, totalCount, originalUrl);
 
-      // we do toStrring on the totalCount because mustache won't bother rendering
+      // we do toString on the totalCount because mustache won't bother rendering
```

Comment:
i sort of intentionally left this alone when i saw it since i value git blame over a decipherable typo. but it's here already.